app/services/trusted_drivers/create_service.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module TrustedDrivers
+  class CreateService < ApplicationService
+    # @attr_reader params [Hash]
+    # - current_user: [User] Current user
+    # - trusted_driver_request: [TrustedDriverRequest] Object request to add how trusted_driver
+
+    def call
+      if trusted_driver_request && receiver_id == current_user.id
+        trusted_driver.save
+        trusted_driver_request.update(accepted: true)
+        OpenStruct.new(success?: true, data: { message: 'created' }, errors: nil)
+      else
+        OpenStruct.new(success?: false, data: nil, errors: 'something went wrong')
+      end
+    end
+
+    private
+
+    def trusted_driver
+      TrustedDriver.new(trusted_driver_id: receiver_id, trust_driver_id: requestor_id)
+    end
+
+    def receiver_id
+      trusted_driver_request&.receiver_id
```

Comment:
look, if `trusted_driver_request` was not passed, this line will return `nil`
the same as a method `requestor_id` below. 
Result will be: 
```ruby
TrustedDriver.new(trusted_driver_id: nil, trust_driver_id: nil)
```
This is not good. Please raise an error, if `trusted_driver_request` was not passed to the service